### ISSUE TITLE
Midi does not work

### ISSUE THREAD

**Issue by Pedro5765** [tsbkelly/Streamdeck-Midibutton#7]
Hi,
I have got this contact from Stream Deck support team.
unfortunately Midi Button does not work on my MacOS system 10.14.6.
I want to send midi notes to switch articulations of virtual instruments in Kontakt Player.
Obviously Stream deck is not recognized as a midi device so that I cannot send any midi notes or commands. If I push the midi note button, I get a yellow error sign. Neither stream deck appears in the Mac Audio MIDI Setup.
Do I have to install anything besides the stream deck installation itself amd the internal Midi button installation?
Versions: Stream Deck software 4.9.2. Midi Button 1.07

Thanks, Peter

**Comment by tsbkelly** [comment]
Hi,

Can you turn on the Debug option in the Midibutton settings, please, and send me the logs? To do this, open Finder, press ⇧⌘G, type ~/Library/Logs/StreamDeck , click Go, and then send me all of the "co.uk.clarionmusic.midibutton" logs, and all of the "StreamDeck" logs.

Thanks

**Comment by Pedro5765** [comment]
Hi,

thanks for Your reply.
I attached he stream deck logs but there are no "co.uk.clarionmusic.midibutton“ logs at all. Should there be a complete folder?

Thanks

**Comment by tsbkelly** [comment]
Hi,

Thanks, but there aren't any Stream Deck logs attached. If there are no Midibutton logs, then the plugin isn't being initialised at all. Can you send me a screenshot of the Stream Deck application showing how you've set up the plugin, please?

Thanks

**Comment by Pedro5765** [comment]
Ok.
I installed Midi button inside the stream deck application under Further actions- see screenshots. Also tried to deinstall and reinstall Midi Button.
If I chose the Use Virtual Port option , no virtual port name appears so I had to manually name it Stream Deck.

Thank You

[stream deck screenshots.zip](https://github.com/tsbkelly/Streamdeck-Midibutton/files/5860521/stream.deck.screenshots.zip)
[streamdeck logs.zip](https://github.com/tsbkelly/Streamdeck-Midibutton/files/5860523/streamdeck.logs.zip)

**Comment by tsbkelly** [comment]
Yes, that's very strange - Stream Deck is loading the plugin, but the plugin isn't being initialised (as otherwise the log would be generated). I'll ask my contact at Elgato if he can shed any light on the situation

**Comment by Pedro5765** [comment]
Ok, thank You and nice evening 

Peter

**Comment by dennispstudio** [comment]
HI

I have the same issue with Midi Button. I can get it to work with Studio One.
I have tried both the virtual port and the non virtual port options. 
Is there a solution? I have the latest version of the Stream Deck XL. 
![Skærmbillede 2021-05-04 kl  11 02 51](https://user-images.githubusercontent.com/83639473/116981829-56862280-acc8-11eb-969a-c110f69591fe.png)
